### PR TITLE
Tips, Potions, Updates and a Fix

### PR DESCRIPTION
**Pull request by kindlich**
Fix: Typo in mkdocs.yml
Updates: Updated some Pages with new entries (e.g. named recipe removal etc)
Potions
And Tipps/Tricks category

### CODE REVIEW

File: docs/Tipps_Tricks/Foreword.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Foreword
+
+This section will contain Tipps and Tricks from experienced Tweakers.
```
Comment (jaredlll08) [inline docs/Tipps_Tricks/Foreword.md]: Typo on "Tips"

File: docs/Tipps_Tricks/General/Loops_Readability.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Use Loops to make your script look better
+
+## Problem
+We all have seen it: Scripts with more than 500 lines where it sais 500 times `recipes.remove(item1);recipes.remove(item2),...`
```
Comment (jaredlll08) [inline docs/Tipps_Tricks/General/Loops_Readability.md]: >sais
"says"

File: docs/Tipps_Tricks/General/Loops_Readability.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Use Loops to make your script look better
+
+## Problem
+We all have seen it: Scripts with more than 500 lines where it sais 500 times `recipes.remove(item1);recipes.remove(item2),...`  
+Not only is this a pain to write, but it is possible that you spend hours debugging a little typo when the only exception you get is `error in recipes.zs: null`
+
+## Solution
+My rule of fist:
```
Comment (jaredlll08) [inline docs/Tipps_Tricks/General/Loops_Readability.md]: rule of thumb

File: docs/Tipps_Tricks/General/Split_Files.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+It is a good idea to split your script into multiple files
+
+## Problem
+- When writing scripts for bigger modpacks, your script might soon become pretty long and confusing.
+- Debugging a long script might take really long, especially if you have an error that doesn't point out a specific line in your script.
+
+## What we know/need to know
+- CraftTweaker can load files from multiple script files.
+- CraftTweaker can even load files in subfolders.
+- Also, CraftTweaker can load .zip files.
```
Comment (jaredlll08) [inline docs/Tipps_Tricks/General/Split_Files.md]: maybe be more specific and say "Crafttweaker can load .zip files that contain scripts inside of them"
also maybe mention that the zip files cannot have a password? I have a feeling someone would ask why their password protected zip isn't being read >.>

File: docs/Tipps_Tricks/Items/IItemDefinition_Metas.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+# Using IItemDefinitions to save a lot of time when referring meta-items
+
+## Problem
+Let's say we want to remove the recipes of some specific colors of wool.  
+White wool has the metadata 0, all colors range from meta 1 to meta 16.
```
Comment (jaredlll08) [inline docs/Tipps_Tricks/Items/IItemDefinition_Metas.md]: meta starts at 0 and goes to 15, so 16 values in total, just starting at 0

File: docs/Vanilla/Recipes/Recipes_Crafting_Table.md [hunk 6]
```diff
@@ -59,6 +59,23 @@ If `wildcard` is true, it will remove shapeless recipes that craft `output` with
 
 Furthermore, `inputs` is optional. If omitted, the function will do the same as `recipe.remove`, though it will only remove shapeless Recipes.
 
+### removeAll
+Rempves all crafting recipes in the game.  
+A bit overkill, don't you think?
+```java
+recipes.removeAll`
```
Comment (jaredlll08) [inline docs/Vanilla/Recipes/Recipes_Crafting_Table.md]:
```
recipes.removeAll()
```

File: mkdocs.yml [hunk 7]
```diff
@@ -68,6 +71,13 @@ pages:
         - Pulverizer: 'Mods/Modtweaker/ThermalExpansion/Pulverizer.md'
         - Refinery: 'Mods/Modtweaker/ThermalExpansion/Refinery.md'
         - Sawmill: 'Mods/Modtweaker/ThermalExpansion/Sawmill.md'
+   - Tipps and Tricks:
+    - Foreword: 'Tipps_Tricks/Foreword.md'
```
Comment (jaredlll08) [inline mkdocs.yml]: "tipps"

File: docs/Tipps_Tricks/General/Split_Files.md (hunk 4 above)
Comment (kindlich) [inline docs/Tipps_Tricks/General/Split_Files.md]: Good point ^^

**Pull request merged**